python/llm/example/GPU/Speculative-Decoding/Eagle/README.md

```diff
@@ -0,0 +1,86 @@
+# Eagle - Speculative Sampling using IPEX-LLM on Intel GPUs
+IPEX-LLM supports EAGLE (Extrapolation Algorithm for Greater Language-model Efficiency) which is a speculative sampling method that improves text generation speed.
+
+See [here](https://arxiv.org/abs/2401.15077) to view the paper and [here](https://github.com/SafeAILab/EAGLE) for more info on EAGLE code.
+
+## Requirements
+To run these examples with IPEX-LLM on Intel GPUs, we have some recommended requirements for your machine, please refer to [here](../../../README.md#requirements) for more information.
+
+### Verified Hardware Platforms
+
+- Intel Data Center GPU Max Series
```

Comment:
Why only Intel Data Center GPU Max Series ?
I have done a simple test on Arc A770, llama2-7b with eagle can run on 16GB Arc A770.
Maybe add `- Intel Arc™ A-Series Graphics and Intel Data Center GPU Flex Series` ?

Reply:
> The examples included in this PR failed consistently on 16GB Arc A770 (out of memory). Can you explain a little more about the test you ran on 16GB Arc A770? For example, what's the script and command-line arguments?

Based on example script, this is my running cmd `python -m evaluation.gen_ea_answer_llama2chat --ea-model-path /home/arda/ruonan/ipex-llm/python/llm/example/GPU/Speculative-Decoding/Eagle/EAGLE-llama2-chat-7B --base-model-path /mnt/disk1/models/Llama-2-7b-chat-hf --enable-ipex-llm`.
And I observed only ~11G memory usage on Arc A770.
`EAGLE-llama2-chat-7B` is downloaded from https://huggingface.co/yuhuili/EAGLE-llama2-chat-7B/tree/main.
I wonder is there anything wrong in my usage ?

Reply:
Your test looks good. I have already added "- Intel Arc™ A-Series Graphics and Intel Data Center GPU Flex" in README. Would you be able to approve and merge? Thanks!